src/main/java/edu/harvard/iq/dataverse/util/FileUtil.java

```diff
@@ -1901,4 +1903,16 @@ public static boolean isFileAlreadyUploaded(DataFile dataFile, Map checksumMapNe
         return false;
     }
 
+    public static String jsonToCSV(JsonArray jsonArray, String... headers) {
```

Comment:
This method doesn't handle arbitrary JSON. Maybe rename to flatJsonToCsv? And add some docs on when to use it?